pom.xml

```diff
@@ -618,5 +646,64 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>format</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>net.revelc.code.formatter</groupId>
+                        <artifactId>formatter-maven-plugin</artifactId>
+                        <version>2.21.0</version>
```

Comment:
I suspect the `<version>` doesn't need to be specified here, as it should pick it up from earlier?

Reply:
removed it